app/presenters/build_presenter.rb

```diff
@@ -0,0 +1,34 @@
+class BuildPresenter
```

Comment:
we have / use serializers in samson ... adding a parallel system looks a bit weird ...